app/grandchallenge/components/views.py

```diff
@@ -153,3 +156,33 @@ def get_success_message(self, cleaned_data):
             success_message=self.success_message,
             url=reverse("notifications:list"),
         )
+
+
+class FileUpdateMixin(ObjectPermissionRequiredMixin, TemplateView):

```

Comment:
Including `TemplateView` here makes it a view, not a `Mixin` (a mixin can be used with any class, this class is limited as it cannot be used with any other view). You could either remove the `TemplateView` from this, or move it to views.py as a base view.

Reply:
Ah true, it's already in views.py, but I will change it's name to `FileUpdateBaseView`